src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimService.java

```diff
@@ -226,33 +222,79 @@ public Claim saveClaim(
             .externalId(externalId)
             .submitterEmail(submitterEmail)
             .createdAt(nowInLocalZone())
-            .letterHolderId(letterHolderId.orElse(null))
+            .letterHolderId(letterHolderId)
             .features(features)
             .claimSubmissionOperationIndicators(ClaimSubmissionOperationIndicators.builder().build())
             .build();
 
         Claim savedClaim = caseRepository.saveClaim(user, claim);
+        String pin = pinResponse.map(GeneratePinResponse::getPin).orElse(null);
+        createClaimEvent(authorisation, user, pin, savedClaim);
+        trackClaimIssued(savedClaim.getReferenceNumber(), savedClaim.getClaimData().isClaimantRepresented());
+
+        return savedClaim;
+    }
+
+    @LogExecutionTime
+    @Transactional(transactionManager = "transactionManager")
+    public Claim saveLegalRepClaim(
+        String submitterId,
+        ClaimData claimData,
+        String authorisation
+    ) {
+        String externalId = claimData.getExternalId().toString();
+        User user = userService.getUser(authorisation);
+        this.verifyUniqueExternalId(user, externalId);
+
+        Optional<GeneratePinResponse> pinResponse = getPinResponse(claimData, authorisation);
+        String letterHolderId = pinResponse.map(GeneratePinResponse::getUserId).orElse(null);
```

Comment:
We don't need letterholder id or pin generated for legal rep claims. This is for citizen claims only.